### ISSUE TITLE
Неверный тип свойства или отсутствие описания типа "char"

### ISSUE THREAD

**Issue by Enotonoff** [bitrix-tools/b24-rest-docs#27]
Здравствуйте. Есть два момента, которые хотелось бы обсудить и при необходимости исправить. В [статье](https://apidocs.bitrix24.ru/api-reference/crm/main-entities-fields.html) описаны поля основных объектов CRM.
1. В статье упоминается тип `char`, описания которого нет в документации. Также при использовании методов вместо этого типа отдается тип `boolean`;
2. При использовании методов, отдающих описания полей, не отдаются поля типа `crm_multifield` (`PHONE`, `EMAIL` и т.д.), зато отдается неизвестное и неописанное поле `fm` этого типа.

Хотелось бы разобраться и понять, почему так происходит и обновить документацию. если необходимо.

**Comment by serg-vostrikov-bitrix** [comment]
Не очень понятно, о чем речь. Поля char отдают y/n, откуда информация о boolean? если это какой-то SDK отдает, то это не вопрос про сам rest. И по второму вопросу тоже непонятно, crm.*.fields отдают crm_multifield.

**Comment by Enotonoff** [comment]
> Не очень понятно, о чем речь. Поля char отдают y/n, откуда информация о boolean? если это какой-то SDK отдает, то это не вопрос про сам rest. И по второму вопросу тоже непонятно, crm.*.fields отдают crm_multifield.

Да, наверное это вопрос к SDK BX24, так как я пользовался методом `BX24.callMethod.crm.item.fields` но в любом случае, я не увидел описания такого типа [здесь](https://apidocs.bitrix24.ru/api-reference/data-types.html).

Если вопросы по SDK нужно обсуждать отдельно, то где?

В целом, на картинке я показал что я имел ввиду под своими вопросами, может так будет нагляднее 
![image](https://github.com/user-attachments/assets/1fa7a7e4-b711-4c7c-a4f2-3237f290a180)

**Comment by Enotonoff** [comment]
А по поводу `boolean` из документации следует, что этот тип тоже может отдавать `Y` или `N` кстати говоря
![image](https://github.com/user-attachments/assets/f59775df-8dda-4877-95c6-cb655b0bff81)

**Comment by serg-vostrikov-bitrix** [comment]
так, ладно. кажется, картинка сложилась. а по поводу обсуждения sdk - это либо в поддержку, либо в чаты с разработчиками в общем порядке

**Comment by Enotonoff** [comment]
> так, ладно. кажется, картинка сложилась. а по поводу обсуждения sdk - это либо в поддержку, либо в чаты с разработчиками в общем порядке

Хорошо, спасибо за обратную связь :)
